resources/lang/en/api.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+	'messages'	=> [
+		'email_voucher_history' => 'Thanks. If you don\'t receive an email with your voucher history, please try again later.'
```

Comment:
@gingerbreadpup Worth noting that the history will be the trader's rather than the user's but I think for the Beta that is mostly the same person?